### DISCUSSION TITLE
Welcome to hermes Discussions!

### DISCUSSION THREAD

**Discussion by joshy** [hermes-router/hermes#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IMAGE-ET** [comment]
Hello, just wanting to say there is a small mistake in your documentation : A session is named "Bookkeeer with Redash", lacking a letter. I hope the best for you. Goodbye

**Reply by tblock79** [reply]
Thanks for pointing that out! Will be fixed in the next version
